mala/common/physical_data.py

```diff
@@ -96,7 +97,10 @@ def read_from_openpmd_file(self, path, units=None):
 
         """
         series = io.Series(path, io.Access.read_only,
-                           options=self.parameters._configuration["openpmd_configuration"])
+                           options=json.dumps(
```

Comment:
In these lines, should we maybe do some `if is string then ... else json.dumps(...)` kind of thing?

Reply:
Wait, no, specifying defaults, which this PR does, relies on the object representation